examples/svm/plot_svm_anova.py

```diff
@@ -20,14 +20,14 @@
 from sklearn.svm import SVC
```

Comment:
Could you please remove this cell and move the imports at the beginning of each cell below: each import should go to the cell it is first needed.

This would make the example nicer by avoiding the wall-of-import effect in the first cell.